recipe/meta.yaml

```diff
@@ -1,15 +1,18 @@
-{% set version = "1.1.0" %}
+{% set name = "hiredis" %}
+{% set version = "2.0.0" %}
+{% set checksum = "81d6d8e39695f2c37954d1011c0480ef7cf444d4e3ae24bc5e89ee5de360139a" %}
+{% set build = "0" %}
 
 package:
-  name: hiredis
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.python.org/packages/source/h/hiredis/hiredis-{{ version }}.tar.gz
-  sha256: 996021ef33e0f50b97ff2d6b5f422a0fe5577de21a8873b58a779a5ddd1c3132
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name|lower }}/{{ name|lower }}-{{ version }}.tar.gz
+  sha256: {{ checksum }}
 
 build:
-  number: 1
+  number: {{ build }}
```

Comment:
Could you add `skip: True  # [py<36]`, see https://github.com/redis/hiredis-py/blob/cca535dc76d4f9380dd2e4214bca73ad826a8387/setup.py#L32 and https://github.com/redis/hiredis-py/blob/v2.0.0/CHANGELOG.md#200-2021-03-28

Reply:
can i ask why py<36 when we only support py>36?

Reply:
If any other developer will build it locally with `py<36` it will work because the upstream source allows it. I think it's good practice not to add unnecessary limitations for users.